tests/scripts/analyze_outcomes.py

```diff
@@ -157,29 +156,30 @@ def read_outcome_file(outcome_file):
                 outcomes[key].failures.append(setup)
     return outcomes
 
-def do_analyze_coverage(outcome_file, args):
+def do_analyze_coverage(results: Results, outcome_file, args):
     """Perform coverage analysis."""
+    results.info("*** Analyze coverage ***")
```

Comment:
Actually I'm not sure about that. Perhaps on top of the 3 basic level info, warning, error, we should have a 4th method like `new_section` or something like that. It would take care of the formatting - for example, `new_section("Analyze coverage")` would end up printing something like `"\n*** Analyze coverage ***\n"`.

On top of the formatting, in the future it would support fancy behaviour like reporting which sections failed at the end etc.

Reply:
Good idea, but I think it's beyond the scope here, so I'd prefer to do it in a follow-up.

Reply:
If it's only for the formatting I think we can address it also here since we're at it. We can just postpone the 2nd part:
> On top of the formatting, in the future it would support fancy behaviour like reporting which sections failed at the end etc.